test-lib.sh

```diff
@@ -837,7 +882,10 @@ EOF
     read -ra mount_options <<< "$(echo "$s2i_args" | grep -o -e '\(-v\)[[:space:]]\.*\S*' || true)"
 
     # Run the build and tag the result
-    docker build ${mount_options[@]+"${mount_options[@]}"} -f "$df_name" --no-cache=true -t "$dst_image" .
+    local id_file
+    mkdir -p "${APP_ID_FILE_DIR:?}"
+    id_file="${APP_ID_FILE_DIR:?}"/"$RANDOM"
+    docker build ${mount_options[@]+"${mount_options[@]}"} --iidfile="$id_file" -f "$df_name" --no-cache=true -t "$dst_image" .
```

Comment:
Let's use podman directly.

Reply:
I think using podman directly needs further discussion also also maybe a separate PR as there could be a lot of changes.

Reply:
we use `docker` everywhere now, so I agree with Zuzka, to keep using it further and if change to podman, then everywhere.